<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VitalStats</class>
 <widget class="QWidget" name="VitalStats">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>419</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
    background: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                                stop:0 lightgreen, stop:1 darkgreen);
}
</string>
  </property>
  <widget class="QLabel" name="VitalStatsLabel">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>20</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt; font-weight:700; font-style:italic;&quot;&gt;Vital Stats Tracker&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="VitalTypeLabel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>80</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Vitals Type: </string>
   </property>
  </widget>
  <widget class="QLabel" name="LastCheckedLabel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>130</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Last Checked: </string>
   </property>
  </widget>
  <widget class="QLabel" name="CheckHomeLabel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>170</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Checked at Home: </string>
   </property>
  </widget>
  <widget class="QComboBox" name="VitalTypeAdd">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>80</y>
     <width>201</width>
     <height>32</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>BMI</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Glucose level</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Blood Pressure</string>
    </property>
   </item>
  </widget>
  <widget class="QDateEdit" name="DateCheckedAdd">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>130</y>
     <width>181</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="CheckAdd">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>160</y>
     <width>201</width>
     <height>32</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Yes</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>No</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="CheckVitalButton">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>220</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Check Vital</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BacktoTabsfromVitals">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>290</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Back to Tabs!</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
